include/rdma/fi_log.h

```diff
@@ -35,8 +35,6 @@
 #ifndef FI_LOG_H
 #define FI_LOG_H
 
-#include "config.h"
```

Comment:
really bad idea: config.h is widely used header which is automatically generated by some autotools. as i can see config.h is not exported on 'make install', and it may conflict with user's configuration

or i missed something?